src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/umbblockgridsortable.directive.js

```diff
@@ -0,0 +1,705 @@
+
+            vm.items = config.items;
+            vm.identifier = config.identifier;
+
+            let scrollElement = null;
+
+            let containerEl = config.containerSelector ? element[0].closest(config.containerSelector) : element[0];
+            if (!containerEl) {
+                console.error("Could not initialize umb block grid sorter.", element[0])
+                return;
+            }
+
+            
+
+            function init() {
+
+                containerEl['umbBlockGridSorter:vm'] = () => {
+                    return vm;
+                };
+                containerEl.addEventListener('dragover', preventDragOver);
+
+                observer.observe(containerEl, {childList: true, subtree: false});
+            }
+            init();
+
+            function preventDragOver(e) {
+                e.preventDefault()
+            }
+
+            function setupItem(element) {
+
+                setupIgnorerElements(element);
+
+                element.draggable = true;
+                element.addEventListener('dragstart', handleDragStart);
+            }
+
+            function destroyItem(element) {
+
+                destroyIgnorerElements(element);
+
+                element.removeEventListener('dragstart', handleDragStart);
+            }
+
+            function setupIgnorerElements(element) {
+                config.ignorerSelector.split(',').forEach(function (criteria) {
+                    element.querySelectorAll(criteria.trim()).forEach(setupPreventEvent);
+                });
+            }
+            function destroyIgnorerElements(element) {
+                config.ignorerSelector.split(',').forEach(function (criteria) {
+                    element.querySelectorAll(criteria.trim()).forEach(destroyPreventEvent);
+                });
+            }
+            function setupPreventEvent(element) {
+                element.draggable = false
+            }
+            function destroyPreventEvent(element) {
+                element.removeAttribute('draggable');
+            }
+
+
+
+
+
+            let currentContainerElement = containerEl;
+            let currentContainerVM = vm;
+
+            let rqaId = null;
+            let currentItem = null;
+            let currentElement = null;
+            let currentDragElement = null;
+            let currentDragRect = null;
+            let dragX = 0;
+            let dragY = 0;
+
+            function handleDragStart(event) {
+
+                if(currentElement) {
+                    handleDragEnd();
+                }
+
+                event.stopPropagation();
+                event.dataTransfer.effectAllowed = "move";// copyMove when we enhance the drag with clipboard data.
+                event.dataTransfer.dropEffect = "none";// visual feedback when dropped.
+                
+                if(!scrollElement) {
+                    scrollElement = getParentScrollElement(containerEl, true);
```

Comment:
It looks like `getParentScrollElement` potentially returns null. Consider if a check is needed for that here as well.

Reply:
Yeah,.. this tries to get the parent scroll element. if none it will return null. which was the value of the prop before.

Its taken care of In the way that where is being used it checks for null. So you can say that null is a valid value. See line 605. :-)